ethers-contract/tests/it/contract.rs

```diff
@@ -1,951 +1,927 @@
-#![allow(unused)]
+use crate::common::*;
```

Comment:
this is a big diff. I'm guessing git is over-reporting a file rearrangement? can you summarize changes?

Reply:
It's 99% indentation, contained only one module